src/encoded/static/components/visualizations.js

```diff
@@ -1,5 +1,63 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { ResultsTable } from './regulome_search';
```

Comment:
It is probably not ideal to be importing components from "regulome_search" into "visualizations" and from "visualizations" into "regulome_search" but I was having trouble thinking of a straightforward way to get around it.